sympy/polys/tests/test_polytools.py

```diff
@@ -3263,7 +3262,7 @@ def test_to_rational_coeffs():
 def test_factor_terms():
     # issue 7067
     assert factor_list(x*(x + y)) == (1, [(x, 1), (x + y, 1)])
-    assert sqf_list(x*(x + y)) == (1, [(x, 1), (x + y, 1)])
+    assert sqf_list((x**2 + 1)  * (x - 1)**2 * (x - 2)**3 * (x - 3)**3) == (1, [(x**2 + 1, 1), (x - 1, 2), (x**2 -5*x + 6, 3)])
```

Comment:
```suggestion
    assert sqf_list((x**2 + 1) * (x - 1)**2 * (x - 2)**3 * (x - 3)**3) == (1, [(x**2 + 1, 1), (x - 1, 2), (x**2 -5*x + 6, 3)])
```

Reply:
Sorry for late response, I was out of town.